### ISSUE TITLE
Can't download .svg icons off /style/iconography/library 

### ISSUE THREAD

**Issue by havesomeleeway** [carbon-design-system/design-system-website#136]
## Detailed description

I'm trying to download the .svg icons off http://carbondesignsystem.com/style/iconography/library but I haven't been able to do so. When I click on the `Download` icon, this page appears. 

![svgfailure](https://user-images.githubusercontent.com/25948209/35074281-17b1165e-fc28-11e7-957c-a383d31a1584.png)

**Comment by tw15egan** [comment]
What browser / OS are you using? Download button works fine for me in Chrome on MacOS

**Comment by havesomeleeway** [comment]
Running on Firefox Quantum 57.0.4 (64-bit). I switched to my Chrome and the download works fine. 

I then did some digging online and realised Firefox has some strict rules about .svg files. 

>Make sure you always declare the namespaces you use in your XML files. If you don't, user agents such as Firefox won't recognize your content and will simply show the XML markup or inform the user that there's an error in the XML. . . .

https://developer.mozilla.org/en-US/docs/Web/SVG/Namespaces_Crash_Course

**Comment by marijohannessen** [comment]
Thanks for looking into this @havesomeleeway ! I can confirm that it does not work in Firefox and that we need to update our SVG's with namespaces. The icons on our website is being pulled from the `carbon-icons` API, so I opened up an issue there so we can track a fix for it https://github.com/carbon-design-system/carbon-icons/issues/54. Closing this issue in favor of that one. Thank you again!
